modules/disk-uploader/pkg/vmexport/vmexport_suite_test.go

```diff
@@ -3,11 +3,14 @@ package vmexport_test
 import (
 	"testing"
 
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/log"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"go.uber.org/zap"
 )
 
 func TestVMExport(t *testing.T) {
+	log.InitLogger(zap.InfoLevel)
```

Comment:
Do we require the logger here?

Reply:
Yes, otherwise the vmexport unit test panics

Reply:
What do you think about including a comment explaining it?

Reply:
Can you add it to the test that requires it?